Package.swift

```diff
@@ -77,7 +77,11 @@ let package = Package(
     ),
     .testTarget(
       name: "CartonTests",
```

Comment:
I don't think `swift test` is actually invoked on CU, is it?

Reply:
There's a "Build and test" workflow but it never actually runs `swift test`, just `carton test` on the test app.

Reply:
Maybe `swift build` in it could be replaced with this?

```
swift test
swift build -c release
```

Using a release build is closer to the real environment I guess. Not sure if `swift test` should use `-c release` too?

Reply:
I'm not sure how big a difference it will make but since release config is closer to the real environment then we should go with that.

Do you want me to add that command in this PR?

Reply:
Please do, it would verify that tests added in this PR run properly on CI.